project_dev_docs/phase1_resources/interview_script.md

```diff
@@ -0,0 +1,44 @@
+# Interview Script for User Research into Open Practices
+
+**Running time:** 30-45 mins
+
+## Context
+
+_What do we want to find out?_
+
+- Interview different people across different themes and at different levels (for example, researcher, project manager, project lead, and so on)
+- Assess the current attitudes towards open working at the Turing
+- Uncover barriers and hidden fears towards open working
+- Understand what Turing people need to address these concerns, what processes need to be put in place, for example
+- Not promoting open working, rather gaining an understanding of the barriers and drivers within the Turing community
+
+## Aims
+
+1. Understand how researchers, theme leaders, project management and research engineers at the Alan Turing Institute define open practices
+2. Understand to what extent these groups are already working in line with open practices in their daily roles
+3. Understand the main barriers and drivers for adoption of working proactices across the Alan Turing Institute
+
+## Questions
+
+1. How would you define the term "open practices" and what does it mean to you in the context of your daily work? [_Answers Aim 1_]
```

Comment:
Sometimes it's helpful to start with an icebreaker question, to ease interviewees into the interview.

- "how did you come to work in this particular role?"
- "what are you excited to be working on at the moment?"
- something else?